task/common/machine/machine-script.sh.tpl

```diff
@@ -13,16 +13,51 @@ sleep 20; while pgrep rclone > /dev/null; do sleep 1; done
 source /opt/task/credentials
 (systemctl is-system-running | grep stopping) || leo stop --cloud="$TPI_TASK_CLOUD_PROVIDER" --region="$TPI_TASK_CLOUD_REGION" "$TPI_TASK_IDENTIFIER";
 END
-
 chmod u=rwx,g=rx,o=rx /usr/bin/tpi-task-shutdown
 
+sudo tee /usr/bin/tpi-task-studio-log << 'END'
```

Comment:
Maybe at least add a few comment explaining what this is for.

It's not like `machine-script.sh.tpl` is  a shining example of literate programming.

Reply:
I think the name suffices `tpi-task-studio-log` however it should be probably `tpi-task-studio-logger`